src/library/scala/util/Using.scala

```diff
@@ -38,15 +41,19 @@ import scala.util.control.{ControlThrowable, NonFatal}
   *
   * Example:
   * {{{
+  * import java.io.{BufferedReader, FileReader}
+  * import scala.util.{Try, Using}
+  *
   * val lines: Try[Seq[String]] = Using.Manager { use =>
   *   val r1 = use(new BufferedReader(new FileReader("file1.txt")))
   *   val r2 = use(new BufferedReader(new FileReader("file2.txt")))
   *   val r3 = use(new BufferedReader(new FileReader("file3.txt")))
   *   val r4 = use(new BufferedReader(new FileReader("file4.txt")))
   *
   *   // use your resources here
-  *   def lines(reader: BufferedReader): Iterator[String] =
-  *     Iterator.unfold(())(_ => Option(reader.readLine()).map(_ -> ()))
+  *   def lines(reader: BufferedReader): Iterator[String] = {
+  *     Iterator.continually(reader.readLine()).takeWhile(_ != null)
+  *   }
```

Comment:
Recommend omitting braces, as it sets a style precedent and adds a line to doc.

Reply:
oh yeah, you're right

Reply:
Fixed